src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputNoWhitespaceBeforeColonOfLabel.java

```diff
@@ -2,13 +2,14 @@
 
 public class InputNoWhitespaceBeforeColonOfLabel {
 
-    {
-        label1 : // violation '':' is preceded with whitespace.'
-        for (int i = 0; i < 10; i++) {}
+  {
+    label1 : // violation '':' is preceded with whitespace.'
+    for (int i = 0; i < 10; i++) {
     }
+  }
 
-    public void foo() {
-        label2:
```

Comment:
Any reason you removed this label?

Reply:
formatter removed it, it also removed the `label1` but it was used to show violation I reverted that. Should I bring back `label2`?

Reply:
I don't see how it is relevant to the current section. I'm ok with the formatter, no actions.

Reply:
same as: https://github.com/checkstyle/checkstyle/pull/15331#discussion_r1688615556

When I formatted the code using vs code, it didn't removed the `label2` but in IntelliJ, it removed it.